app/views/rmt_view.py

```diff
@@ -32,10 +32,11 @@ def get_emp_info(name, on_false={}, on_missing='Unspecified'):
 
     emp_job_role = {
             'Job Role ID': current_job_role['uniqueRoleId'],
-            'Badge Number': get_emp_info('idBadgeNo'),
-            'Postcode': get_emp_info('postcode'),
+            'Job Role': current_job_role['jobRole'],
             'Job Role Short': current_job_role['jobRoleShort'],
             'Job Role Type': current_job_role['jobRoleType'],
+            'Badge Number': get_emp_info('idBadgeNo'),
+            'Postcode': get_emp_info('postcode'),
```

Comment:
I'm not sure why these were added, they're not mentioned in any ticket I can find.

Reply:
I`m not too sure either :| 
Have removed